hardhat.config.ts

```diff
@@ -39,16 +39,15 @@ const buildForkConfig = (): HardhatNetworkForkingUserConfig => {
       blockNumber: 15003356,
     };
   }
-  console.log(forkMode);
   return forkMode;
 };
 
 const getTenderlyForkConfig = (): string => {
-  let url = JSON.parse(readFileSync("tenderlyConfig.json").toString());
+  // let url = JSON.parse(readFileSync("tenderlyConfig.json").toString());
```

Comment:
fix tenderly fork generation later

Reply:
yeah we can scrap that and replace url later